domain-model/src/main/java/uk/gov/hmcts/cmc/domain/constraints/ValidCountyCourtJudgmentValidator.java

```diff
@@ -20,7 +20,7 @@ public void initialize(ValidCountyCourtJudgment constraintAnnotation) {
 
     @Override
     public boolean isValid(CountyCourtJudgment ccj, ConstraintValidatorContext context) {
-        if (ccj == null) {
+        if (!isEligibleForThisValidator(ccj)) {
```

Comment:
That does not read too well. Why not simple `isEligible` which wold be sufficient in this context.

Reply:
I don't agree.

```java
    @Override
    public boolean isValid(CountyCourtJudgment ccj, ConstraintValidatorContext context) {
        if (!isEligible(ccj)) {
            return true;
        }
```

Intent of this check is to see if this validator has enough information to be able to apply it's validation or rely on other validators to apply theirs. `isEligible` does not reveal this intent at all.